src/index.html

```diff
@@ -1,13 +1,308 @@
+                class="testimonial__image"
+              >
+              <h3 class="testimonial__name">Travel</h3>
+              <div class="testimonial__info">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </div>
+            </div>
+
+            <div class="testimonial">
+              <img
+                src="./images/photos/t3.png"
+                alt="Virtual group tours"
+                class="testimonial__image"
+              >
+              <h3 class="testimonial__name">Virtual group tours</h3>
+              <div class="testimonial__info">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="Contacts">
+        <div class="container">
+          <h2 class="page__section--title">Contact us</h2>
+          <div class="contacts__content">
+            <div class="contacts__info">
+              <div class="contacts__info--first-museum">
+                <h4 class="contacts__name">The Met Fifth Avenue</h4>
+                <div class="contacts__sample">phone</div>
+                <div class="contacts__number">+1 212-535-7710</div>
+                <div class="contacts__sample">address</div>
+                <div class="contacts__street">
+                  1000 Fifth Avenue <br> New York, NY 10028
+                </div>
+
+              </div>
+              <div class="contacts__info--second-museum">
+                <h4 class="contacts__name">The Met Cloisters</h4>
+                <div class="contacts__sample">phone</div>
+                <div class="contacts__number">+1 212-923-3700</div>
+                <div class="contacts__sample">address</div>
+                <div class="contacts__street">
+                  99 Margaret Corbin Drive <br> Fort Tryon Park <br> New York, NY 10040
+                </div>
+              </div>
+            </div>
+            <form action="#" class="contacts__feedback">
+              <input
+                name="name"
+                type="text"
+                placeholder="NAME"
+                class="contacts__form"
+              >
+
+              <input
+                name="email"
+                type="email"
+                placeholder="EMAIL"
+                class="contacts__form"
+              >
+
+              <textarea
+                name="message"
+                placeholder="MESSAGE"
+                class="contacts__form contacts__form--textarea"
+              ></textarea>
+
+              <div class="button">
+                <a href="#" class="button--link">send</a>
+              </div>
+            </form>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer"></footer>
```

Comment:
footer picture should stretch all the way
![image](https://user-images.githubusercontent.com/37161315/170642873-28a156e5-4d3c-407c-807e-c3708e2a0cd8.png)